configuration-service/deploy/service.yaml

```diff
@@ -60,7 +60,7 @@ spec:
             memory: "256Mi"
             cpu: "500m"
           limits:
-            memory: "256Mi"
+            memory: "512Mi"
```

Comment:
Not directly related to the Issue, but I increased this value because I kept getting OOM errors when trying to attach the debugger to the container.